spec/spec_helper.rb

```diff
@@ -11,4 +11,8 @@
   end
 
   config.shared_context_metadata_behavior = :apply_to_host_groups
+
+  config.after :all do
+    ApplicationRecord.subclasses.each(&:delete_all)
```

Comment:
Why??

Reply:
The users that FactoryBot creates during the test stay in the test database.
When Rspec is run again, FactoryBot can't create the same user again because user already exist, Error: This email has been used by another user.
This code erase test database after each Rspec run.

Reply:
Okay. Let’s go with that for now.

Reply:
After using let() I didn't need this code, so removed it.